src/main/scala/com.gu.gibbons/services/BonoboService.scala

```diff
@@ -10,22 +10,22 @@ import model._
 trait BonoboService[F[_]] {
 
   /** Get all the users which have been created, or which
-   * have been extended, before `jadis`
+   * have been extended, before `date`
    *
-   * @param jadis  The time before which a user is
+   * @param date  The date before which a user is
    *               potentially expired
    */
-  def getUsers(jadis: Instant): F[Vector[User]]
+  def getUsers(date: Instant): F[Vector[User]]
```

Comment:
@rowannekabalan I think `date` is not a great name because you lose the intent (`before` vs `now`) and dual encode the type. Please note as well [`Date` and `Instant` are 2 different things](https://stackoverflow.com/questions/32437550/whats-the-difference-between-instant-and-localdatetime) so it will be quite confusing for someone reading the code.

Reply:
Thanks @mchv! This change is actually outdated as I quickly realised "date" is quite ambiguous so I ended up naming the variables (creationDate) and (reminderDate) depending on the context. But thanks for pointing out the Date vs Instant confusion, I will again rename them to something better with the new French knowledge that I've gained :)